_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: OTel trace group
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OTel trace group
+
+The `OTel trace group` processor completes missing trace group related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend. The `OTel trace group` processor identifies the missing trace group information for a `spanId` by looking up the relevant fields in its root `span` stored in OpenSearch.
+
+## OpenSearch
+
+See the following example configuration for `OTel trace group` processor when using self-managed OpenSearch:
```

Comment:
> when using self-managed OpenSearch

Perhaps this should be re-worded some. This is really for when you use username/password authentication with OpenSearch.

Perhaps:

> when connecting to an OpenSearch cluster using username and password.